src/etcd/lease.cr

```diff
@@ -15,8 +15,11 @@ class Etcd::Lease
 
   # Requests persistence of lease.
   # Must be invoked periodically to avoid key loss.
-  def keep_alive(id : Int64)
+  def keep_alive(id : Int64) : Int64?
     Model::KeepAlive.from_json(client.api.post("/lease/keepalive", {ID: id}).body).result
+  rescue e : Exception
```

Comment:
This should be a JSON parse exception

Reply:
Nope it's a pure Exception class atm. It should be a JSON parse exception though
https://github.com/crystal-lang/crystal/blob/cc22dc0682d69f9d79617b16af71d480a987adc0/src/json/pull_parser.cr#L385